app/components/tooltip-renderer.js

```diff
@@ -1,25 +1,20 @@
 import Component from '@ember/component';
 import { argument } from '@ember-decorators/argument';
-import { defineProperty, computed as computedProperty } from '@ember/object';
-import Ember from 'ember';
+import { computed } from '@ember-decorators/object';
+import mustache from 'mustache';
 
 export default class TooltipRenderer extends Component {
-  init(...args) {
-    super.init(...args);
-
-    this.setProperties(this.get('feature.properties'));
-
-    defineProperty(this, 'layout', computedProperty(() => {
-      const template = this.get('template');
-      const { properties } = this.get('feature');
-
-      return Ember.HTMLBars.compile(template, properties);
-    }));
-  }
-
   @argument
   feature = {}
 
   @argument
   template = ''
+
+  @computed('feature', 'template')
+  get renderedText() {
+    const properties = this.get('feature.properties');
+    const template = this.get('template');
+
+    return mustache.render(template, properties);
+  }
 }
```

Comment:
Nice! I prefer HTMLBars because it comes with Ember but mustache is fine.

The biggest thing here is we got rid of this v dangerous thing: `this.setProperties(this.get('feature.properties'));`.

Reply:
Okay, maybe that `this.setProperties` was causing the bigger issue? For some reason, I wasn't able to log `Ember.HTMLBars.compile(template, properties)` and assumed that was causing the problem. So I just made the decision to refactor it to look similarly to our other apps where map tooltips were working. Also wanted to up our consistency with mustache since we were using it in zola. But, again, not 100% sure what was causing the actual bug! Maybe at some point, when we have time, we can refactor all of the `tooltip-renderer` components to use HTMLBars, OR maybe we can add it to labs-ui as a reusable component since we reuse it in several of our apps

Reply:
I'm not sure either but one problem is that the application doesn't control the received keys in `feature.properties` so something important might've been getting replaced.

If you wanted, I would try to see if the HTMLBars.compile in your new renderedText there.

Reply:
Okay, I did a little trial by replacing that `mustache.render` with `Ember.HTMLBars.compile`  and it looks like the HTMLBars breaks it with an error `idFn is not a function`. So looks like HTMLBars is the problem after all

Reply:
Good to know. Thanks for checking that out :)